src/main/java/seedu/address/model/person/predicates/AvailableTimePredicate.java

```diff
@@ -8,29 +8,43 @@
  * Tests that a {@code Person}'s {@code FreeTime} is within the FreeTime that is given.
  */
 public class AvailableTimePredicate implements FindCommandPredicate {
+    private static final String[] DAY_OF_WEEK = {"Monday", "Tuesday", "Wednesday", "Thursday", "Friday"};
```

Comment:
Consider using something like this `DayOfWeek.of(i + 1).getDisplayName(TextStyle.SHORT, Locale.ENGLISH);`